lib/Sema/TypeCheckType.cpp

```diff
@@ -2428,7 +2428,7 @@ Type TypeResolver::resolveAttributedType(TypeAttributes &attrs,
         // a function type, we'll remove it.
         if (i == TAK_escaping) {
           diag.fixItRemove(getTypeAttrRangeWithAt(Context,
-                                                  attrs.getLoc(TAK_escaping)));
+                                                attrs.getLoc(TAK_escaping)));
```

Comment:
Oops, not this one though, line 2421. Thanks!

Reply:
I'd suggest running clang-format on your changes @ismetanin

Reply:
Thanks for your suggestion, now everything should be ok (or at least clang-formatted)

Reply:
It seems like you've only run it on the last commit.

Reply:
Looks like you're right. I've run clang-format over all my changes now
Sorry for these formatting mistakes, now I know how to properly format code

Reply:
No worries, thanks for being patient!

Reply:
Maybe would be good if those indentation changes, could be squashed in a single commit?

Reply:
@xedin can squash them upon merging

Reply:
No problems, I've squashed indentation changes with the second commit